src/server/HelixServer.js

```diff
@@ -127,6 +133,9 @@ export default class HelixServer extends EventEmitter {
   async start() {
     const { log } = this;
     if (this._port !== 0) {
+      if (this._kill) {
+        await fetch(`http://localhost:${this._port}/.kill`);
+      }
```

Comment:
I would not do this pre-emptively, but rather after checking `inUse`. and you might need to wait a little for the other process to stop and release the socket.